specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/networkInterface.json

```diff
@@ -897,10 +897,34 @@
         "provisioningState": {
           "$ref": "./network.json#/definitions/ProvisioningState",
           "description": "The provisioning state of the network interface IP configuration."
+        },
+        "privateLinkConnectionProperties": {
+          "$ref": "#/definitions/NetworkInterfaceIPConfigurationPrivateLinkConnectionProperties",
+          "description": "PrivateLinkConnection properties for the network interface."
         }
       },
       "description": "Properties of IP configuration."
     },
+    "NetworkInterfaceIPConfigurationPrivateLinkConnectionProperties": {
```

Comment:
Addition of a new property should involve a new api-version.  See https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?e=xEcppo  (specifically the "New property added to response" example).

Reply:
This is a new API version, `2019-07-01`, isn't not merged into master yet, and there are no SDKs published that use it

Reply:
ah, didn't realize this was into a feature branch